source/aws-connect-vm-serverless/src/main/java/com/amazonaws/kvstream/KVSProcessRecordingLambda.java

```diff
@@ -65,14 +73,32 @@ private boolean processData(String data) {
             return false;
         }
 
+        AmazonDynamoDBClientBuilder builder = AmazonDynamoDBClientBuilder.standard();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you don’t explicitly set an AWS region, the SDK consults the default region provider chain to try and determine the region to use.  However, the resulting delay might lead to a poor cold-start. If code portability is not an issue, consider explicitly setting the AWS region using the *withRegion()* method.

[Learn more](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#automatically-determine-the-aws-region-from-the-environment)